src/lib.rs

```diff
@@ -79,12 +97,12 @@ impl RobloxStudio {
     #[must_use]
     #[inline]
     pub fn exe_path(&self) -> PathBuf {
```

Comment:
You don't have to do this, but maybe I should change the name of this to `application_path`!